pkg/util/git.go

```diff
@@ -0,0 +1,20 @@
+package util
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+// GetRemoteAndRepo splits a refspec string (eg, jenkins-x/jjx) into the two strings for the origin (jenkins-x)
+// and repo (jx)
+func GetRemoteAndRepo(refspec string) (origin, repo string, err error) {
```

Comment:
I think, this function can be moved into `git` package.

Reply:
see `gits.ParseGitURL` https://github.com/jenkins-x/jx/blob/d4775203bb5b42eed07dc6b5f8cb31f4a2a5ec2a/pkg/gits/git_url.go#L101 and https://github.com/jenkins-x/jx/blob/b079e7c82ea4b03732156a3e2fce86ee45ea1ece/pkg/gits/git_url_test.go